packages/@interaxyz/mobile/src/onboarding/success/OnboardingSuccessScreen.tsx

```diff
@@ -19,11 +20,27 @@ function OnboardingSuccessScreen() {
 
   const { t } = useTranslation()
 
+  const assetsConfig = getAppConfig().themes?.default?.assets
+
+  const image =
```

Comment:
Could this just be `const image = assetsConfig?.onboardingSuccessImage`? Is there a reason to prefer this pattern?

Reply:
This would be so a builder could use different assets for `light | dark | ...-mode`.

Reply:
How do you mean @MuckT ? I don't see how that works.

I don't think there is any significance to the pattern. I think its a leftover from when assetsConfig type was any

Reply:
I went with the pattern @jophish suggested

Reply:
That makes sense. I thought the discussion was about putting the assets at a higher level, such as in `getAppConfig()?.assets`.